client/src/modules/stock/movements/templates/action.cell.html

```diff
@@ -1,4 +1,4 @@
-<div class="ui-grid-cell-contents text-action" uib-dropdown dropdown-append-to-body uib-dropdown-toggle>
+<div class="ui-grid-cell-contents text-action" uib-dropdown dropdown-append-to-body uib-dropdown-toggle ng-if="!row.groupHeader">
```

Comment:
As you suggested in the email, this enhancement should done in very registry with grouping option. So an issue should be better.